src/ipld/util.rs

```diff
@@ -293,13 +304,28 @@ pub fn stream_chain<DB: Blockstore, T: Iterator<Item = Tipset> + Unpin>(
     db: DB,
     tipset_iter: T,
     stateroot_limit: ChainEpoch,
-) -> impl Stream<Item = anyhow::Result<Block>> {
+) -> ChainStream<DB, T> {
     ChainStream {
         tipset_iter,
         db,
         dfs: VecDeque::new(),
         seen: CidHashSet::default(),
         stateroot_limit,
+        fail_on_dead_links: true,
+    }
+}
+
+pub fn stream_graph<DB: Blockstore, T: Iterator<Item = Tipset> + Unpin>(
```

Comment:
Any way to indicate `fail_on_dead_links: false` in the function name or doc? e.g. `stream_graph_unstrict`

Reply:
I've updated the documentation of both `stream_graph` and `stream_chain`.